### PR TITLE
Provide hints on how to suppress s2i behavior

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: assemble [hunk 1]
```diff
@@ -73,7 +73,7 @@ function restore_lock() {
           thamos log
         fi
 
-        echo ">>> Thoth stack analysis failed, this build will fail shortly..." >&2
+        echo ">>> Thoth stack analysis failed, this build will fail shortly; you can suppress this faulure by providing THOTH_ERROR_FALLBACK=1" >&2
```
Comment (KPostOffice) [inline assemble:76]: *failure

**Review by KPostOffice — COMMENTED** [review]
Just one small spelling issue

File: assemble (hunk 1 above)
Comment (fridex) [reply assemble]: Thanks, adjusted in 195ebc0

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**